Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -1,5 +1,7 @@
-import demistomock as demisto  # noqa: F401
-from CommonServerPython import *  # noqa: F401
+register_module_line('Panorama', 'start', __line__())
+### pack version: 2.1.3
```

Comment:
what's this?

Reply:
My bad, I copied the code out of the UI. Changing back.